src/keycloak/chart/templates/pvc.yaml

```diff
@@ -22,8 +82,15 @@ metadata:
     {{- include "keycloak.labels" . | nindent 4 }}
 spec:
   accessModes:
-    - ReadWriteOnce
+    - {{ .Values.persistence.accessMode | quote }}
   resources:
     requests:
-      storage: 512Mi
+      storage: {{ .Values.persistence.themes.size | quote }}
+  {{ if .Values.persistence.storageClassName }}
+  {{- if (eq "-" .Values.persistence.storageClassName) }}
+  storageClassName: ""
+  {{- else }}
+  storageClassName: "{{ .Values.persistence.storageClassName }}"
+  {{- end }}
```

Comment:
Think I'm a bit confused on the template here - what is the intent of the `-` check? `storageClassName: ""` would be equivalent to no storage class I believe? Is no storage class an actual need here, or can we just simplify the template to:
```suggestion
  {{ if .Values.persistence.storageClassName }}
  storageClassName: "{{ .Values.persistence.storageClassName }}"
  {{- end }}
```

Reply:
Yep - was tracking the difference just wasn't sure if we needed to add that support here (we already assume storage class for other charts that I don't think support this). All good with leaving though if you think it's a value add.

Reply:
It's probably overkill in this case.. #deletedeletedelete 😉